app/frontend/src/components/Inputs/Chip.tsx

```diff
@@ -7,52 +7,66 @@ import { combineClasses } from "../Utility/utils";
 
 interface ChipProps {
   addClass?: string;
-  isActive?: boolean;
-  onChange: (active: boolean, value: string) => any;
+  checked?: boolean;
+  onClick?: (active: boolean, value: string) => any;
```

Comment:
Thank you! I had always struggled with using any, since it felt so wrong, but I could not logically restrict the return arbitrarily.

I had looked it up and found the [unknown type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type). Thoughts on this? I'm leaning towards this as it still promotes type safety.

Reply:
As for your other point on the chip, the designers seem to have indicated that it is a single choice chip, but this was not defined as a variation in the design system. I'm not sure I need to make a variant of it. I did, however, create a `.fill` class for scenarios where widths needed to fit the container they were in, for whatever reason. I'll add it to the demo, so let me know if you find this alternative aaceptable.